hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/wal/TestAsyncLogRolling.java

```diff
@@ -49,6 +61,122 @@ public static void setUpBeforeClass() throws Exception {
     conf.setInt(FanOutOneBlockAsyncDFSOutputHelper.ASYNC_DFS_OUTPUT_CREATE_MAX_RETRIES, 100);
     conf.set(WALFactory.WAL_PROVIDER, "asyncfs");
     AbstractTestLogRolling.setUpBeforeClass();
+
+    // For slow sync threshold test: roll once after a sync above this threshold
+    TEST_UTIL.getConfiguration().setInt(FSHLog.ROLL_ON_SYNC_TIME_MS, 5000);
+  }
+
+  @Test
+  public void testSlowSyncLogRolling() throws Exception {
```

Comment:
Since we move some method from FSHLog to AbstractFSWAL, is it possible to just move the testSlowSyncLogRolling to AbstractTestLogRolling, instead of writing a new one here?